cmd/tdexd/main.go

```diff
@@ -113,11 +115,20 @@ func main() {
 
 	log.Info("starting daemon")
 
+	statsDir := filepath.Join(config.GetString(config.DataDirPathKey), "stats")
+	ctx, cancelStats := context.WithCancel(context.Background())
+	stats.EnableMemoryStatistics(
```

Comment:
shoudl this be something we enable in sort of debug mode or we want it always on?

Reply:
If we only use Debugf then there is no sense in waiting for resources, so yes I will add this check.

Reply:
Done